backend/test/api/scores_test.clj

```diff
@@ -0,0 +1,55 @@
+(ns api.scores-test
+  (:require [expectations.clojure.test :refer :all]
+            [api.base :as api]
+            [test-helper :as helper]
```

Comment:
once again, we need a more descriptive name than `helper` (could be another issue)

Reply:
I suggest "auto-play" for that file, instead of `test-helper`